odgi viz binned mode, color by bin mean coverage, or by bin mean inversion rate

AndreaGuarracino opened the pull request:
**Yeast pangenome**
Color by the bin mean coverage:

`odgi viz -i Sc+Sp.pan.fpal10k.sYgs.odgi -o Sc+Sp.pan.fpal10k.sYgs.c.png -b -w 1000 -g -c`

![Sc+Sp.pan.fpal10k.sYgs.c](https://user-images.githubusercontent.com/62253982/90402915-745f1900-e0a0-11ea-8ab9-24e5000020bc.png)


Color by the bin mean inversion rate:

`odgi viz -i Sc+Sp.pan.fpal10k.sYgs.odgi -o Sc+Sp.pan.fpal10k.sYgs.z.png -b -w 1000 -g -z`

![Sc+Sp.pan.fpal10k.sYgs.z](https://user-images.githubusercontent.com/62253982/90402930-79bc6380-e0a0-11ea-8ee4-459e6b275eb1.png)

subwaystation commented:
I just tested the whole package of possible visualizations again for DRB1-3123. 

Default.
![DRB1-3123 fa gz gfa og Y og](https://user-images.githubusercontent.com/10184961/90407169-77f59e80-e0a6-11ea-834d-ba091a21d2f0.png)


New `color-by-mean-inversion`.
![DRB1-3123 fa gz gfa og Y og zb](https://user-images.githubusercontent.com/10184961/90406521-95763880-e0a5-11ea-9de2-dfdd57a777ff.png)

Old `show-strand`. Wondering if we still need this? @AndreaGuarracino ?
![DRB1-3123 fa gz gfa og Y og S](https://user-images.githubusercontent.com/10184961/90406609-b76fbb00-e0a5-11ea-95a9-d2e069d49063.png)

New `color-by-mean-coverage`.
![DRB1-3123 fa gz gfa og Y og cb](https://user-images.githubusercontent.com/10184961/90406828-faca2980-e0a5-11ea-8b6e-fb1410867caf.png)

Old `change-darkness` + `white-to-black`
![DRB1-3123 fa gz gfa og Y og d u](https://user-images.githubusercontent.com/10184961/90407081-5399c200-e0a6-11ea-86dc-eb45e1d693d8.png)

Old `change-darkness`
![DRB1-3123 fa gz gfa og Y og d](https://user-images.githubusercontent.com/10184961/90407130-67452880-e0a6-11ea-8763-ee09e0e69ef0.png)

AndreaGuarracino commented:
With the `show-strand` option, a path is colored respect to the winner between #nodes forward vs #nodes reverse, so it has always one color. Now each bin has its own color respect to the coverage or the inversion rate. I updated the doc to be more clear and explicit.

I also fixed a problem in the coverage normalization, using the max bin mean coverage. Now the new `color-by-mean-coverage` option looks (correctly) like
![ciao](https://user-images.githubusercontent.com/62253982/90506450-97e39b80-e154-11ea-9ecd-be40b2acd1fd.png)

We can see a little bin where the paths go several times.

ekg commented:
This looks awesome. I just tried it on the LPA VNTR:

![SD_for_EG edyeet p80a95 2 smooth og cov](https://user-images.githubusercontent.com/145425/90533399-b9567e80-e178-11ea-92a9-388d01e89255.png)

Makes it perfectly obvious where the CNV is! (Well, in odgi viz space... it'd be great to annotate these with text / numbers somehow...)

AndreaGuarracino commented:
It should close the #32 issue.
